lib/hacluster.pm

```diff
@@ -718,8 +719,9 @@ sub check_cluster_state {
         $cmd->("$crm_mon_cmd | grep -i 'no inactive resources'");
     }
     $cmd->('crm_mon -1 | grep \'partition with quorum\'');
```

Comment:
eventually mind to add a UT about `check_cluster_state` in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/t/11_hacluster.t

Reply:
here a starting point https://github.com/os-autoinst/os-autoinst-distri-opensuse/compare/master...mpagot:os-autoinst-distri-opensuse:check_cluster_state_ut

Reply:
Done.